src/Targets/Microsoft.CSharp.DesignTime.targets

```diff
@@ -43,4 +43,19 @@
       <Context>Project</Context>
     </PropertyPageSchema>
   </ItemGroup>
+  
+  <!-- Targets -->
+
+  <!-- Returns Csc command-line arguments for the language service -->
+  <Target Name="CompileDesignTime"
+          Returns="@(CscCommandLineArgs)"
+          DependsOnTargets="Compile"
+          >
+
+    <Error Condition="'$(SkipCompilerExecution)|$(ProvideCommandLineArgs)' == 'true|true'"
```

Comment:
Is the condition correct? Shouldn't it error in the opposite case when they aren't true|true?